===================================================================
RCS file: /horde/cvs/imp/compose.php,v
retrieving revision 2.1093
retrieving revision 2.1094
diff -p --unified=3 -r2.1093 -r2.1094
--- imp/compose.php	2008/05/25 16:33:29	2.1093
+++ imp/compose.php	2008/05/26 11:51:45	2.1094
@@ -111,7 +111,9 @@ $thismailbox = Util::getFormData('thisma
 require_once 'Horde/Token.php';
 if (isset($conf['token'])) {
     /* If there is a configured token system, set it up. */
-    $tokenSource = Horde_Token::factory($conf['token']['driver'], $conf['token']);
+    $tokenSource = Horde_Token::factory(
+        $conf['token']['driver'],
+        Horde::getDriverConfig('token', $conf['token']['driver']));
 } else {
     /* Default to the file system if no config. */
     $tokenSource = Horde_Token::factory('file');
